packages/contracts/src/test/osx-versions/Migration.sol

```diff
@@ -0,0 +1,23 @@
+// SPDX-License-Identifier: AGPL-3.0-or-later
+
+pragma solidity 0.8.17;
```

Comment:
After a call with @Rekard0, a reminder for us to create a follow-up so that the osx-version package can deal with differing compiler versions in the legacy files @brickpop

Reply:
OS-431 created :+1: